docker-compose.yml

```diff
@@ -90,6 +90,7 @@ services:
     volumes:
       - "./src/aws/lambda-complete:/var/task/lambda-complete"
       - "./src/aws/utils:/var/task/utils"
+      - "./src/.prettierrc:/var/task/.prettierrc"
```

Comment:
Question not directly related to this work: why not moving prettier configuration in the project's root instead?

Reply:
We can but I see no benefit.

Reply:
Pardon me, but putting it in `src` is a bit like "tomber comme un cheveu sur la soupe" to me...

![](https://media.giphy.com/media/x6q2SEFflggiQ/giphy.gif)

Reply:
I really don't like eating a soup with an hair in it.